2_process/src/summarize_targets.R

```diff
@@ -1,5 +1,5 @@
 summarize_targets <- function(ind_file, ...) {
```

Comment:
I would recommend naming an argument instead of `...` here for clarity as we're only expecting a single input (something like `target_names`).